firmware/throttle/src/throttle_control.cpp

```diff
@@ -141,6 +109,7 @@ void enable_control( void )
         && g_throttle_control_state.operator_override == false )
     {
         const uint16_t num_samples = 20;
+        //TODO: Should this be inside the cli()?!?
```

Comment:
Probably not? If I understand correctly the only source of interrupt is brake reports, and that shouldn't affect the voltages on the signal input lines. I think it's safe to take a few voltage samples, be interrupted by the report timer, resume and get the rest of the voltage samples, apply them to the disconnected DAC, and then toggle the spoof relay to connect the spoof signals.

The only place where I think we might need to disable interrupts is when we're actually setting the DAC outputs in `oscc_dac.cpp`, but that might fall outside of this discussion. And as noted earlier the DAC isn't connected to the signals at this point so it's not a huge deal.

Reply:
Wow shouldn't have left the `TODO:` in there but really good points for further analysis of the DAC behavior. I'll remove the `TODO:` for now since we documented these comments internally.